Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/ParticleSmearing.cpp

```diff
@@ -101,11 +101,19 @@ ActsExamples::ProcessCode ActsExamples::ParticleSmearing::execute(
       params[Acts::eBoundLoc0] = sigmaD0 * stdNormal(rng);
       params[Acts::eBoundLoc1] = sigmaZ0 * stdNormal(rng);
       params[Acts::eBoundTime] = time + sigmaT0 * stdNormal(rng);
-      // smear direction angles phi,theta ensuring correct bounds
-      const auto [newPhi, newTheta] = Acts::detail::normalizePhiTheta(
-          phi + sigmaPhi * stdNormal(rng), theta + sigmaTheta * stdNormal(rng));
-      params[Acts::eBoundPhi] = newPhi;
-      params[Acts::eBoundTheta] = newTheta;
+      // smear direction angles phi, theta ensuring correct bounds
+      while (true) {
```

Comment:
This could result in an endless loop if `theta` is very shallow to begin with and `sigmaTheta` is small. Again I would argue that it is the user's responsibility to provide valid parameters so we do not need to worry something falls apart afterwards.

Reply:
If we put it on the user, then `sigmaTheta = 0` is the only safe value for them to provide, due to the randomness.

Reply:
But that is a limitation of the propagation / fitter. This code produces valid bound track parameters so I would not change it.